### PR TITLE
Add remove list tasks

### PR DESCRIPTION
**Pull request by zunairkhan811**
The project is completed according to the following instruction:
Remove all hardcoded items from the tasks array.
Create a new JavaScript file for the new functionality.
Implement a function for adding a new task (add a new element to the array).
Implement a function for deleting a task (remove an element from the array).
Implement a function for editing task descriptions.
By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..
All changes to the To Do List should be saved in local storage.

### CODE REVIEW

**Review by fobadara — COMMENTED** [review]
Hi @zunairkhan811, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that contains the live demo section in its README file but doesn't have a working live demo link.  Kindly add a working link or remove this section if it is not required

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/previous.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+}, {
+  description: 'complete to do list project',
+  completed: 'false',
+  index: '1',
+}];
+
+arr.forEach((items) => {
+  taskList.innerHTML += `
+    <div class="tasks"><input type="checkbox" name="first" value="first" id="first"><label class="tasks-text" for="first">${items.description}</label><span class="tasks-icon"><i class="fa fa-ellipsis-v" aria-hidden="true"></i></span></div>`;
+});
```
Comment (GzimAsani) [inline src/previous.js]: - [ ] Please remove the previous hardcoded array. I understand that you are not using it but you code will look unprofessional if you have unused code there
image

File: src/index.js [hunk 2]
```diff
@@ -1,18 +1,93 @@
...
+      el.addEventListener('dblclick', (e) => {
+        e.target.disabled = true;
+        doticon[i].style.display = 'block';
+        removeiconId[i].style.display = 'none';
+        editTask();
+        savelocal(textareaInput[i].value, i);
+      });
+    });
+  });
+}
```
Comment (GzimAsani) [inline src/index.js]: - [x] Based on your UI I am unable to edit a task. I think for better user experience when clicking enter or outside the box, the task should get updated/edited. BTW your edit code looks fine :+1: https://www.awesomescreenshot.com/video/16122594?key=560650414e9a64331c8ec122ac688ddc

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @zunairkhan811 a               :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the local storage and add/remove functions  :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @zunairkhan811, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- ADD, EDIT, and DELETE functionalities working as expected :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**